examples/sensor-websocket/src/components/SensorWebSocket.jsx

```diff
@@ -0,0 +1,206 @@
+import React, { Component } from "react";
+import {
+  connection_trigger_payload,
+  disconnection_trigger_payload,
+  getChannelToken,
+  getDeviceDataById,
+  getWSUrl,
+  socketParams,
+  value_trigger_payload
+} from "../apiHandler";
+
+import { Alert, Col, Container, ListGroup, Row } from "react-bootstrap";
+import moment from "moment";
+
+require("moment/min/locales.min");
```

Comment:
We use because we need to find user language, 
according to your reference `event.toLocaleString('en-GB', { timeZone: 'UTC' })`
How I will get 'en-GB' because I think we need to dynamic, that's why I use above library for this to get user language and make locale datetime format in that language.

Reply:
What about navigator.language?
https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language

Reply:
Yes, We are using navigator.language.
This is the code 
```
require("moment/min/locales.min");

function getLocaleFormat() {
    const locale = window.navigator.language;
    moment.locale(locale);
    return moment.localeData().longDateFormat("LLL");
}
```
In Moment js, we can't update locale without this js `moment/min/locales.min` 
mention in docs https://momentjs.com/docs/#/use-it/browserify/

Reply:
Then you have the answer to your previous question

Reply:
Yes, I'm already using this in all examples.

Reply:
How's that required when you can simply do as follows?
```
const locale = window.navigator.language;
event.toLocaleString(locale, { timeZone: 'UTC' })
```

Reply:
You can also retrieve the timezone by the way

Reply:
Okay let me do this.